### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by meetshah02** [tijilicious/Wine-quality-prediction-and-validation-#1]
Hi, the Fractal test has a condition that if the quality is less than 6 it should be bad, equal to 6 should be normal and more than 6 is good.
How did you tackle that ? Can you share the exact code you used ??

**Comment by tijilicious** [comment]
You can define a function to do that. Also i don't have the actual code this is just for the reference to help. Try using hyper parameter tuning for actual test. That will improve accuracy.

I have made a little change in code and added the quality printing function. You can refer to that.

**Comment by meetshah02** [comment]
Tried using function for that but didn't found any success with accuracy. Can you tell the function code you used ?? Also which hyperparameter technique did you use; I tried RandomizedSearchCV. Also I am attaching below screenshots of my code. If you could review it and help me out in this it would be really great.
![IMG_20210505_131131](https://user-images.githubusercontent.com/37037060/119926157-75f53000-bf94-11eb-911a-a1d772b9d6a7.jpg)
![IMG_20210505_131136](https://user-images.githubusercontent.com/37037060/119926176-7e4d6b00-bf94-11eb-952a-119e8fe08259.jpg)
![IMG_20210505_132747](https://user-images.githubusercontent.com/37037060/119926188-83121f00-bf94-11eb-8c05-fac29652b89b.jpg)
![IMG_20210505_132759](https://user-images.githubusercontent.com/37037060/119926202-87d6d300-bf94-11eb-91bb-4af72b01045e.jpg)
![IMG_20210505_132808](https://user-images.githubusercontent.com/37037060/119926211-8c9b8700-bf94-11eb-8d2e-ffbb7d894cfb.jpg)
![IMG_20210505_132816](https://user-images.githubusercontent.com/37037060/119926218-90c7a480-bf94-11eb-978e-bdde2aa0bd30.jpg)

**Comment by tijilicious** [comment]
I do not remember the question exactly, try using random forest if allowed in the question. Along with randomizedsearchCV. I will try to provide code for that soon.

**Comment by meetshah02** [comment]
I have attached pictures of the question above. If referring to that you can provide a code it would be of great help. 
Until that can you keep the issue open so that I would know that you are working on it ? Awaiting your code.

**Comment by tijilicious** [comment]
Please check the hyper parameter training part. It is still at a low bias but the variance is high. This could be due to having lesser data to work with since i could only find the single data file and in the question it gives you separate files to train, test and validate. You can try changing the hyperparameters by referring to the sklearn documentation for random forest regressor.
Hope that helps. 
I will keep the issue open till Sunday in case if you want some more help regarding this.
